src/components/link/Link.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import './Link.scss';
+
+export default function Link({
+  href,
+  text,
+  external,
+}: {
+  href: string;
+  text: string;
+  external: boolean;
```

Comment:
Tän external propsin vaikutukset on ehkä hieman implisiittisiä. Mietin olisko joku toinen nimi kuvaavampi? Tyyliin: 'openToNewTab' tjsp.?

Reply:
My idea behind this was that we could have different default behavior (and maybe design) for internal and external links.

Reply:
Yeah I understood that. I think it would be ok if that default behaviour could be easily derived from the api (either by explanation or variable name). But not sure how to do that. My fear is, that to understand what is happening, developers would have to dig in deeper to know what the effect of having external link is. Instead of just looking at the Link api (props), and seeing directly from there how the Link component is going to behave based on props.

Reply:
What if we create separate components: Link and ExternalLink? That would reduce the amount of conditions and emphasize more that we have two types of links hmm

Reply:
I'll go with separate ExternalLink 👍